src/addons/meta/cursor.c

```diff
@@ -109,7 +109,8 @@ int32_t get_elem_count(
     const EcsOpaque *opaque = scope->opaque;
 
     if (scope->vector) {
-        return ecs_vec_count(scope->vector);
+        //TODO: getting size from get_elem_count looks strange
```

Comment:
This doesn't look right, the `get_elem_count` function shouldn't return the vector size.